cmd/prometheus/main.go

```diff
@@ -541,6 +544,16 @@ func main() {
 			level.Warn(logger).Log("msg", "Time retention value is too high. Limiting to: "+y.String())
 		}
 
+		if cfg.tsdb.MaxPercentage > 100 {
+			cfg.tsdb.MaxPercentage = 100
```

Comment:
Wdyt about log maybe a warning saying the operator provided a percentage gt 100 and this is not possible, so the default 100 will be used?